common/locales/en-US/main.ftl

```diff
@@ -125,6 +125,7 @@ friends = Friends
     .blocked-desc = Blocked
     .add-existing-friend = You are already friends!
     .add-failed = No such user was found!
+    .add-yourself = Can't add yourself, silly goose!
```

Comment:
we have a key named `cannot-add-self` would that work as well? do we need both keys? 🔨